e2e/app/page/old-cdr-version-modal.ts

```diff
@@ -0,0 +1,76 @@
+import {Page} from 'puppeteer';
+
+import Button from 'app/element/button';
+import Modal from 'app/component/modal';
+import {waitWhileLoading} from 'utils/waits-utils';
+import {ElementType} from '../xpath-options';
+import {LinkText} from '../text-labels';
+import Checkbox from '../element/checkbox';
+
+const LabelAlias = {
+    WILL_USE_OLD_CDR_VERSION: 'I will use this workspace to complete an existing study or replicate a previous study.',
+    WILL_IDENTIFY_OLD_CDR_VERSION: 'In the workspace description below, I will identify which study I am continuing or replicating.',
+}
+
+const FIELD = {
+    willUseCheckbox: {
+        textOption: {name: LabelAlias.WILL_USE_OLD_CDR_VERSION, type: ElementType.Checkbox}
+    },
+    willIdentifyCheckbox: {
+        textOption: {name: LabelAlias.WILL_IDENTIFY_OLD_CDR_VERSION, type: ElementType.Checkbox}
+    },
+    continueButton: {
+        textOption: {name: LinkText.Continue}
+    },
+}
+
+export default class OldCdrVersionModal extends Modal {
+
+    constructor(page: Page, xpath?: string) {
+        super(page, xpath);
+    }
+
+    async isLoaded(): Promise<boolean> {
+        const xpath = '//*[@data-test-id="old-cdr-version-modal"]';
+        await Promise.all([
+            this.page.waitForXPath(xpath),
+            waitWhileLoading(this.page),
+        ]);
+        return true;
+    }
+
+    async getWillUseCheckbox(): Promise<Checkbox> {
+        return Checkbox.findByName(this.page, FIELD.willUseCheckbox.textOption);
```

Comment:
nit: finding any object in a modal should use the modal parent: `Checkbox.findByName(this.page, FIELD.willUseCheckbox.textOption, this);`